config/sample_manifest_excel/columns.yml

```diff
@@ -870,3 +870,9 @@ control_type:
   conditional_formattings:
     empty_cell:
     is_error:
+priority:
+  heading: PRIORITY
+  unlocked: true
+  conditional_formattings:
+    empty_cell:
+    number_greater_than_zero:
```

Comment:
Does this enforce it to be greater than zero in the excel file? Zero is a valid priority isn't it? (no priority)